src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1725,10 +1725,12 @@ internal enum ErrorCode
         ERR_NotNullConstraintMustBeFirst = 8713,
         WRN_NullabilityMismatchInTypeParameterNotNullConstraint = 8714,
 
-        ERR_DuplicateNullSuppression = 8715
+        ERR_DuplicateNullSuppression = 8715,
 
         #endregion diagnostics introduced for C# 8.0
 
+        ERR_ReAbstractionInNoPIAType = 8750,
+
```

Comment:
I think this belongs above the #endregion marker above, as it is being introduced for C# 8.0.

Reply:
> I think this belongs above the #endregion marker above, as it is being introduced for C# 8.0.

Moved

---
In reply to: [315921033](https://github.com/dotnet/roslyn/pull/38140#discussion_r315921033) [](ancestors = 315921033)